_maps/RandomRuins/SpaceRuins/chaosmarine.dmm

```diff
@@ -0,0 +1,598 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"a" = (
+/turf/open/space/basic,
+/area/space)
+"b" = (
+/turf/closed/indestructible/cult,
+/area/ruin/space/has_grav/powered)
+"c" = (
+/obj/effect/portal/permanent/one_way{
+	color = "#FF3333";
+	desc = "Going inside might be a REALLY bad idea.";
+	id = "chaos_marine_enter";
+	light_color = "#FA8282";
+	light_power = 3;
+	light_range = 2;
+	name = "dark portal"
+	},
+/turf/open/indestructible/cult,
+/area/ruin/space/has_grav/powered)
+"d" = (
+/obj/effect/decal/cleanable/blood,
+/obj/structure/destructible/cult/pylon{
+	corrupt_delay = 1e+031;
+	desc = "A floating crystal intended to increase the power of those who server Blood God.";
+	last_corrupt = 1e+031
+	},
+/turf/open/indestructible/cult,
+/area/ruin/space/has_grav/powered)
+"e" = (
+/obj/structure/chair/bronze{
```

Comment:
alt+clicking on this chair will produce a message that refers to ratvarian technology

Reply:
Yeah, it's somewhat fine.